### ISSUE TITLE
Odd syslog message (release 3.3)

### ISSUE THREAD

**Issue by moodeaudio** [mikebrady/shairport-sync#859]
Hi,

Some moOde users report seeing the following syslog message even though the option in the conf file is correct.

syslog
```
shairport-sync: Invalid disable_standby_mode option choice "uto". It should be "always", "auto" or "never". It is set to "never".
```

/etc/shairport-sync.conf
```
disable_standby_mode = "while_active"; // Some DACs make small "popping" noises when they go in and out of standby mode. This prevents entry to standby mode. Settings can be: "always", "while_active" or "never". Default is "never", but only for backwards compatibility. You can use "yes" instead of "always" and "no instead of "never".
```

**Comment by mikebrady** [comment]
Hi Tim. Thanks for the post. The actual issue is that I've updated this setting – "while_active" is no longer a valid option – it was replaced by the option "auto". So the message *should* read:

```
shairport-sync: Invalid disable_standby_mode option choice "while_active". It should be "always", "auto" or "never". It is set to "never".
```
I'll fix it. A 3.3.1 is on its way...

**Comment by mikebrady** [comment]
Hello again – just pushed a 3.3.1rc1 release with that problem and a few others fixed.

**Comment by moodeaudio** [comment]
Cool, thanks :-)

**Comment by mikebrady** [comment]
Fixed in 3.3.1.
